### DISCUSSION TITLE
Allow you to adjust the size of the collision/physics layer editor window in Tilemaps

### DISCUSSION THREAD

**Discussion by Tolly24** [godotengine/godot-proposals#6060] [2D]
I recently upgraded to beta and been working on my tilesets for a 2D pixel art project. First of all I have to say that the new tilemap is amazing and such an improvement from earlier versions. I a problem adding collision polygons to a tilemap in the "physics layer" section where the window to add and edit them is seriously tiny. Adding collisions in such a tiny window is only usable because you can zoom in so much and for someone with not the best eyesight, I need to blow it up to 2016% just to put down a uniform collision shape. It adds a lot of tedium and is just really uncomfortable to work with when you know you need to add collisions to a fresh tileset. In fact, I had to resort to drawing guides in a bright color in my sprite editor and re-export the tileset without the guides once I've made all the polygons. If I don't, it's simply impossible(for me at least) to add uniform shapes that wont result in undesired edges across similar tiles. 

If anyone has a better way to do it, please let me know. Currently, it's only possible to resize the width of the window and this is the biggest I can make the window for my setup(I use a 1440p monitor, might not be as big of an issue on 1080p). ![image](https://user-images.githubusercontent.com/65282021/211163148-b315a764-bd5f-4294-81db-09f0f131a6bc.png)

**Comment by peti29** [comment]
So much this!

My tiles are 12 by 10 pixels.... press an 'F' for me!

I'll attach a screenshot. There's plenty of space in all directions to the physics shape editor but it just wouldn't grow.
An ever better solution could be an option to open the editor in a separate window.

![editor](https://github.com/godotengine/godot-proposals/assets/70334968/1af8c62c-53f4-4320-b12b-6f27e6befb51)
